### PR TITLE
[binance] Fixed json compression error

### PR DESCRIPTION
**Pull request by Hypeon**
See issue https://github.com/bitrich-info/xchange-stream/issues/170

### CODE REVIEW

**Comment by Flemingjp** [comment]
@Hypeon any chance of updating this PR so that is compiles?

File: xchange-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingService.java [hunk 1]
```diff
@@ -67,5 +68,8 @@ public ProductSubscription getProductSubscription() {
         return productSubscription;
     }
 
-
+    @Override
+    protected WebSocketClientExtensionHandler getWebSocketClientExtensionHandler() {
+        return null;
```
Comment (badgerwithagun) [inline xchange-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingService.java]: Should this be `BinanceWebSocketHandler.INSTANCE`?  Just based on https://github.com/sanderdg/xchange-stream/commit/0e2dd0866f09f8a6cecc0531ca84e264523c81b8 which is the Bitmex version of this fix

Comment (Flemingjp) [reply xchange-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingService.java]: This seems to be the case

Comment (badgerwithagun) [reply xchange-binance/src/main/java/info/bitrich/xchangestream/binance/BinanceStreamingService.java]: So is this a mistake in the pull request?

Is the pull request actually needed for Binance?

It does sound like it's needed for Bitmex...

**Comment by Flemingjp** [comment]
@badgerwithagun You're referencing https://github.com/sanderdg/xchange-stream/commit/0e2dd0866f09f8a6cecc0531ca84e264523c81b8 right?

**Comment by badgerwithagun** [comment]
Yep

**Comment by Flemingjp** [comment]
Closing as its unclear what this PR is trying to fix. To maintain the prior fix to BitMex, I have opened #233

**Comment by Hypeon** [comment]
@Flemingjp Sorry, that i haven't answered earlier, but github mails were in spam. 

I wasn't clear, but this error occurs for both Bitmex and Binance module. I have just implemented the same thing for Binance.

**Pull request closed without merging**